### DISCUSSION TITLE
Inconsistencies with Fluent Design and WinUI 3

### DISCUSSION THREAD

**Discussion by Poopooracoocoo** [microsoft/fluentui#26215] [General]
Resources used:
[WinUI 3 Gallery](https://apps.microsoft.com/store/detail/winui-3-gallery/9P3JFPWWDZRC)
[Fluent UI React demo](https://react.fluentui.dev/)

System info: 
Windows 11 22H2

Note:
I will not cover hover or pressed states unless mentioned but it's _very_ easy to see the differences for yourselves.

Why this is an issue:
As per @micahgodbolt:
>Fluent v8 is based on the previous design iteration. We're working on a v9 set of controls that use the new language.

**Radio button:**
Expected (WinUI 3): 
<img width="74" alt="image" src="https://user-images.githubusercontent.com/51363864/211035677-520e2211-4c47-4b5a-9dfc-11b3efe9cc89.png">

Actual (Fluent UI React v9):  
<img width="252" alt="image" src="https://user-images.githubusercontent.com/51363864/211036100-01a454b8-a06a-4225-a628-67e63830605f.png">

**Checkbox:**
Expected:
<img width="129" alt="image" src="https://user-images.githubusercontent.com/51363864/211036689-4fced19e-721b-441d-b5a4-e01c4b35f475.png">

Actual:
<img width="80" alt="image" src="https://user-images.githubusercontent.com/51363864/211036644-ea116879-483d-46a0-acfe-1e8864c171e1.png">

**Button:**
Expected:
<img width="131" alt="image" src="https://user-images.githubusercontent.com/51363864/211036919-c5d5867a-e649-4c49-91a2-6076819b30e4.png">

Actual:
<img width="92" alt="image" src="https://user-images.githubusercontent.com/51363864/211036890-aad3d44e-8caa-4835-a8f9-59f469dfab59.png">

**Dialog:**
Expected:
<img width="253" alt="image" src="https://user-images.githubusercontent.com/51363864/211037154-08a5b3d3-ae21-4e38-97ad-be6d99973220.png">

Actual:
<img width="466" alt="image" src="https://user-images.githubusercontent.com/51363864/211037076-9330a01b-80fb-409c-89f1-5b3a8a7a7df9.png">

**Slider:**
Expected:
<img width="166" alt="image" src="https://user-images.githubusercontent.com/51363864/211037558-12297f86-b45e-45d0-ab0f-bc95d0f7f1bc.png">

Actual:
<img width="103" alt="image" src="https://user-images.githubusercontent.com/51363864/211037508-973aeea5-fd01-4cce-a18e-081f48553db0.png">

**Switch (pressed state):**
Expected:
<img width="63" alt="image" src="https://user-images.githubusercontent.com/51363864/211037897-db93038c-04ed-438a-9ed8-8244182c927d.png">

Actual:
<img width="122" alt="image" src="https://user-images.githubusercontent.com/51363864/211037980-958c363f-4b36-46e2-ad39-a03861fe0b34.png">

**Menu (hover state):**
Expected:
<img width="99" alt="image" src="https://user-images.githubusercontent.com/51363864/211038379-f34be776-6dbf-4e95-98f1-e0820044294e.png">

Actual:
<img width="122" alt="image" src="https://user-images.githubusercontent.com/51363864/211038430-f84f96ea-2ab1-4b30-8c6a-4451a583a753.png">

**Editable combobox (hover state):**
Expected:
<img width="158" alt="image" src="https://user-images.githubusercontent.com/51363864/211038685-1faf36bb-fd95-40de-965a-e7315f3089ba.png">

Actual:
<img width="201" alt="image" src="https://user-images.githubusercontent.com/51363864/211038718-11646ab4-05a2-4b83-8953-7572198fa0a1.png">

**Comment by Biyakuga** [comment]
There has still been no official explanation for why this is the case. This example appears to be only the tip of the iceberg, there are many other inconsistencies between WinUI 3 and Fluent UI React that lack any clear or well justified rationale
